frontend/sass/styles.scss

```diff
@@ -323,3 +323,19 @@ html.jf-is-fullscreen-admin-page {
         display: none;
     }
 }
+
+.jf-moratorium-banner {
+    .container {
+        .jf-close-button {
+            z-index: 30;
+            cursor: pointer;
+            
+            &:hover {
+                opacity: 0.7;
+            }
+        }
```

Comment:
As I mentioned in an earlier comment, Bulma does have a `.delete` class and it'd be nice to use that if possible, both to maintain visual consistency and avoid CSS bloat, but it's up to you!

Reply:
👍